app/src/main/java/org/fossasia/openevent/general/attendees/AttendeeFragment.kt

```diff
@@ -26,20 +27,18 @@ import org.koin.android.architecture.ext.viewModel
 class AttendeeFragment : Fragment() {
 
     private lateinit var rootView: View
-    private val EVENT_ID: String = "EVENT_ID"
-    private val TICKET_ID: String = "TICKET_ID"
     private var id: Long = -1
     private val attendeeFragmentViewModel by viewModel<AttendeeViewModel>()
-    private lateinit var ticketId: TicketId
     private lateinit var eventId: EventId
+    private var ticketIdAndQty: ArrayList<Pair<Int, Int>>? = null
```

Comment:
Type should be of List

Reply:
These are standard Java guidelines, never make a variable of implementation, always of interface